src/packsquash.js

```diff
@@ -0,0 +1,59 @@
+import { endGroup, exportVariable, getBooleanInput, startGroup, warning } from '@actions/core';
+import { exec } from '@actions/exec';
+import { rmSync } from 'fs';
+import { addProblemMatcher, removeProblemMatcher } from './problem-matcher';
```

Comment:
I'd rename the `problem-matcher` module to `problem_matcher` to be more consistent.

```suggestion
import { addProblemMatcher, removeProblemMatcher } from './problem_matcher';
```

Reply:
Fix by c7f34d2e8097b62ba4dc97c22d86fb667c5f3cbf